executor/executor_pkg_test.go

```diff
@@ -61,3 +68,81 @@ func (s *testExecSuite) TestMergeHandles(c *C) {
 		c.Assert(kr.EndKey, DeepEquals, ekr.EndKey)
 	}
 }
+
+// mockSessionManager is a mocked session manager which is used for test.
```

Comment:
Can this mocked sessionManager be moved into util/mock package so that it can be resued?

Reply:
But this is only useful in this package. Is there any other package need to use it?

Reply:
We could move it out to mock package when we meet the scenario.